examples/advanced/llm_hf/sft_job.py

```diff
@@ -73,10 +73,16 @@ def main():
         job.to(dequantizer, "server", tasks=["train"], filter_type=FilterType.TASK_RESULT)
 
     # Define the model persistor and send to server
-    # First send the model to the server
-    job.to("src/hf_sft_model.py", "server")
-    # Then send the model persistor to the server
-    model_args = {"path": "src.hf_sft_model.CausalLMModel", "args": {"model_name_or_path": model_name_or_path}}
+    if train_mode.lower() == "sft":
+        # First send the model to the server
+        job.to("src/hf_sft_model.py", "server")
+        # Then send the model persistor to the server
+        model_args = {"path": "src.hf_sft_model.CausalLMModel", "args": {"model_name_or_path": model_name_or_path}}
+    elif train_mode.lower() == "peft":
```

Comment:
minor thing, should this job call "sft_job.py"?

Reply:
good point! updated to llm_hf_fl_job